src/constants/translations/en/edit-tutor-profile.json

```diff
@@ -6,6 +6,6 @@
     "profile": "Profile",
     "professionalInfo": "Professional info",
     "notifications": "Notifications",
-    "passwordSecurity": "Password & Security"
+    "passwordAndSecurity": "Password & Security"
```

Comment:
check `passwordSecurityTab` - `Password & Security` is already exist property. 
Maybe to pass some values to the `common` and reuse it ?

Reply:
I will better fix it in #1928